### DISCUSSION TITLE
Log Drains Private Alpha

### DISCUSSION THREAD

**Discussion by Ziinc** [supabase/supabase#26650] [Changelog]
Log drains is currently private alpha, and is available for Teams and Enterprise customers. We are still firming up the [pricing and documentation](https://github.com/supabase/supabase/pull/26370), however it will likely involve a flat fee and variable egress usage. This will be announced separately through official channels.

We will be supporting **Datadog** as our initial provider. 

The following destinations are in the works:
1. Elastic/Filebeat
2. Syslog

We are currently onboarding interested customers manually, so please fill out this form to get started: https://forms.supabase.com/logdrains.

**Comment by MildTomato** [comment]
@Ziinc might need to update the text above slightly, this text gets published on the [changelog page](https://supabase.com/changelog). It says `"cc @encima"`

**Reply by Ziinc** [reply]
Done

**Comment by thenbe** [comment]
Hi. Would [vector](https://vector.dev/) help you implement this feature?

For example, if you've already implemented syslog output then, using vector, one can translate a [syslog](https://vector.dev/docs/reference/configuration/sources/syslog/) source to any of these available [sinks](https://vector.dev/docs/reference/configuration/sinks/) (including [elasticsearch](https://vector.dev/docs/reference/configuration/sinks/elasticsearch/)).

**Reply by Ziinc** [reply]
We are in the midst of implementing a [syslog drain](https://github.com/Logflare/logflare/pull/2067).

We did not go the vector wrapping route (though this was considered) as dispatching of log events is not the only concern of logflare as the key analytics engine of the Supabase stack.

**Comment by landsman** [comment]
Integration with [Betterstack](https://betterstack.com/telemetry) would be nice!
